vignettes/downloading_a_long_species_list.Rmd

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Don't assume things I did were correct :) feel free to change anything

Reply:
Yes I thought it was a bit weird too but I just kept it in case there was some unintended consequences to removal. I think I will just leave it for now, since it's been that way...